apps/devhub/tests/test_views.py

```diff
@@ -1124,9 +1125,32 @@ def get_step(self):
         return SubmitStep.objects.get(addon=self.get_addon())
 
 
-class TestSubmitStep1(TestSubmitBase):
+class TestAgreement(TestSubmitBase):
+    def test_agreement_first(self):
```

Comment:
Since that the agreement page had to be abstracted into a separate page we can't mock the submit function. We pass the parameter of the page to actually render into a separate function and then mock it in both this function and the one that tests submit but at that point I think it ends up making the tests confusing to the point of where it isn't worth it.